src/components/Landing/Overview.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import {Link} from 'react-router-dom'
+import axios from 'axios'
+
+import "./Overview.css"
+
+export default function Overview() {
+
+  const [numBlocks, setNumBlocks] = useState(0)
+  const [numTransactions, setNumTransactions] = useState(0)
+
+  useEffect(() => {
+    axios.get('http://127.0.0.1:8000/blocks/length').then(resp => {
```

Comment:
use env file to store urls or any other config variables

Reply:
done